client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SkipNotificationsFlagTest.java

```diff
@@ -69,6 +69,10 @@ public void testPutAll() {
       performTest(RequestType.PUT_ALL);
    }
 
+   public void testRemoveValue() {
```

Comment:
for some reason, I don't think this change belongs to this PR... :)

Reply:
a test in a wrong branch